src/bin/deku_cli.ml

```diff
@@ -486,17 +475,19 @@ let withdraw_proof node_folder operation_hash callback =
       |> String.concat " ;\n"
       |> String.trim);
     await (`Ok ())
+
+let folder_dest =
+  let docv = "folder_dest" in
+  let doc =
+    "The folder the files will be created in. The folder must exist and be \
+     empty." in
+  let open Arg in
+  required & pos 0 (some string) None & info [] ~doc ~docv
+
```

Comment:
I would even
1. regroup folder_dest and folder_node (and similar declarations) together in the file
2. factorize them in a function, which could help giving more precise help messages for each
command

Am I missing something, or does folder_dest (which describes a destination folder, which must be
empty) only make sense for the setup_tezos command and not for withdraw_proof, which doesn't write
any file but just prints stuff on stdout?

Reply:
I think `folder_dest` can just be deleted and replaced with `folder_node`. I made a commit that does this here: https://github.com/marigold-dev/deku/pull/580/commits/2a3eede547190ae1ae8fe2980f9018f328f2df72; @ulrikstrid @aguillon feel free to revert if you don't like this change.